src/routes/PageRoutes.js

```diff
@@ -2,6 +2,7 @@ import { Home } from "../pages/Home";
 import { Login } from "../pages/Login";
 import { Signup } from "../pages/Signup";
 import { ForgotPassword } from "../pages/ForgotPassword";
+import { ProductListing } from "../pages/ProductListing";
```

Comment:
same suggestion all pages can be imported in one index.js (for that particular folder) and exported from there